### PR TITLE
Display notice about GA4

### PR DESCRIPTION
**Pull request by felixarntz**
## Summary

Addresses issue #2219 

**Do not merge until final copy and link URL has been added (see issue ACs).**

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by felixarntz** [comment]
@aaemnnosttv This has the final copy and appearance. The notices are styled in a slightly inconsistent way now, with warning looking like it did before, info looking like in the mock and suggestion looking like before except having its colors adjusted to match the info colors (which were almost the same).

Since we're on a tight schedule here, we can revisit this holistically another time.

Relevant Storybook:
* https://google.github.io/site-kit-wp/storybook/pull/2222/?path=/story/global-notices--settings-warning-notice
* https://google.github.io/site-kit-wp/storybook/pull/2222/?path=/story/global-notices--settings-info-notice
* https://google.github.io/site-kit-wp/storybook/pull/2222/?path=/story/global-notices--settings-suggestion-notice
* https://google.github.io/site-kit-wp/storybook/pull/2222/?path=/story/analytics-module-setup--start

File: stories/notices.stories.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import SettingsNotice, { TYPE_WARNING, TYPE_INFO, TYPE_SUGGESTION } from '../assets/js/components/settings-notice';
+
+global._googlesitekitLegacyData.canAdsRun = true;
```
Comment (aaemnnosttv) [inline stories/notices.stories.js:11]: Was this copied over from another stories file? I don't think it's relevant to notices.

File: stories/notices.stories.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/**
```
Comment (aaemnnosttv) [inline stories/notices.stories.js:1]: This file is missing a file header.

**Review by aaemnnosttv — COMMENTED** [review]
Looks good to me @felixarntz - just a few very minor details in storybook.

**Pull request merged**